meta_request/test/unit/meta_request/utils_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+describe MetaRequest::Utils do
+  describe ".dev_callsite" do
+    it "returns line parsed with Callsite if rails root is found in multi-line trace" do
+      stacktrace = [
+        "/gem/gem_file.rb:1:in `func'`",
+        "#{File.join(MetaRequest.rails_root, "test_file.rb")}:87:in `app_func'",
+        "/gem/gem_file.rb:1:in `func2'`",
+      ]
+      expected_callsite_line = Callsite::Line.new("/Users/jan/gat/rails_panel/meta_request/test_file.rb", 87, "app_func")
```

Comment:
Would this test even work on other machines due to this absolute path?

Reply:
🤦‍♂ thanks for catching that obvious bug